testing/trino-testing/src/main/java/io/trino/testing/AbstractTestEngineOnlyQueries.java

```diff
@@ -6049,6 +6049,18 @@ public void testExplainExecuteWithUsing()
         assertEquals(getOnlyElement(result.getOnlyColumnAsSet()), getExplainPlan("SELECT * FROM orders WHERE orderkey < 7", LOGICAL));
     }
 
+    @Test
+    public void testExplainExecuteWithShow()
```

Comment:
There should be a test where the values node row count is smaller than the truncation limit.
There should also be a test for the empty rows case.
We should also have assertion for rowCount in the explain output